doc/Release_Build_Creation.rst

```diff
@@ -8,7 +8,7 @@ This file consists of the procedure that should be followed to generate the rele
 Procedure
 ***************
 
-#. Setup a CentOS 7.8.2003 system.
+#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7/isos/x86_64/>`_ system.
```

Comment:
@justinzw Above links shows to `CentOS 7.9.2009` ISO files . For `CentOS 7.8.2003` - We need to use https://vault.centos.org/7.8.2003/isos/x86_64/

Reply:
Good catch! thanks for seeing that. I updated the link to 7.8.2003.

Thanks @shailesh-vaidya